dynatrace-aws-logs.sh

```diff
@@ -22,7 +22,7 @@ function print_help_main_options {
   echo ""
   printf \
 "usage: dynatrace-aws-logs.sh [-h]
-                              {deploy,subscribe,unsubscribe,discover-log-groups}
+                              {deploy,delete,subscribe,unsubscribe,discover-log-groups}
```

Comment:
I think delete is more like last option :)

Reply:
N/A